src/OrchardCore.Modules/OrchardCore.Rules/Drivers/JavascriptConditionDisplayDriver.cs

```diff
@@ -32,8 +57,39 @@ public override async Task<IDisplayResult> UpdateAsync(JavascriptCondition condi
             var model = new JavascriptConditionViewModel();
             await updater.TryUpdateModelAsync(model, Prefix);
 
-            // TODO is empty.
-            condition.Script = model.Script;
+            if (string.IsNullOrWhiteSpace(model.Script))
```

Comment:
Let's move this general comment above the whole code block where it makes sense.

```suggestion
            // CodeMirror hides the textarea which displays the error when updater.ModelState.AddModelError() is used,
            // that's why a notifier is used to show validation errors.
            if (string.IsNullOrWhiteSpace(model.Script))
```